doc/lightningd-config.5.txt

```diff
@@ -0,0 +1,302 @@
+
+*bitcoin-rpcport*='PORT'::
+    The bitcoind(1) RPC port to connect to.
+
+*default-fee-rate*='SIPA'::
+    Satoshis per kw if bitcoind can't estimate fees.
+
+*rescan*='BLOCKS'::
+    Number of blocks to rescan from the current head, or absolute blockheight
+    if negative.  This is only needed if something goes badly wrong.
+
+Lightning daemon options:
+
+*lightning-dir*='DIR'::
+    Sets the working directory. All other files are relative to this.
+
+*pid-file*='PATH'::
+    Specify pid file to write to.
+  
+*log-level*='LEVEL'::
+    What log level to print out: options are io, debug, info, unusual, broken.
+
+*log-prefix*='PREFIX'::
+    Prefix for log lines: this can be customized if you want to merge logs with
+    multiple daemons.
+
+*log-file*='PATH'::
+    Log to this file instead of stdout.
+
+*rpc-file*='PATH'::
+    Set JSON-RPC socket (or /dev/tty), such as for lightning-cli(1).
+
+*daemon*::
+    Run in the background, suppress stdout and stderr.
+
+Lightning node customization options:
+
+*rgb*='RRGGBB'::
+    Your favorite color as a hex code.
+
+*alias*='NAME'::
+    Up to 32 UTF-8 characters to tag your node.  Completely silly, since anyone
+    can call their node anything they want.  The default is an
+    NSA-style codename derived from your public key, but "Peter Todd"
+    and "VAULTERO" are good options, too.
+
+*fee-base*='MILLISATOSHI'::
+    The base fee to charge for every payment which passes through.  Note that
+    millisatoshis are a very, very small unit!
+
+*fee-per-satoshi*='MILLIONTHS'::
+    This is the proportional fee to charge for every payment which passes
+    through.  As percentages are too course, it's in millionths, so 10000
+    is 1 percent, 1000 is 0.1%.
+
+*ignore-fee-limits*='BOOL'::
+    Allow nodes which establish channels to us to set any fee they
+    want.  This may result in a channel which cannot be closed, should
+    fees increase, but make channels far more reliable since we never
+    close it due to unreasonable fees.
+
+*commit-time*='TIME'::
+    How long to wait before sending commitment messages to the peer: in
+    theory increasing this would reduce load, but your node would have to be
+    extremely busy node for you to even notice.
+
+Lightning channel and HTLC options:
+
+*locktime-blocks*='BLOCKS'::
+    How long we need to spot an outdated close attempt, which is also
+    how long the peer would need to wait if they perform a unilateral
+    close.
+
+*max-locktime-blocks*='BLOCKS'::
+    The longest we'll ever allow a peer to hold up payments, in the worst
+    case.  If they ask for longer, we'll refuse to create a channel,
+    and if an HTLC asks for longer, we'll refuse it.
+
+*funding-confirms*='BLOCKS'::
+    Confirmations required for the funding transaction when the other side
+    opens a channel before the channel is usable.
+
+*commit-fee-min*='PERCENT'::
+*commit-fee-max*='PERCENT'::
+    Limits on what onchain fee range we'll allow when a node opens a
```

Comment:
Is this also a percentage of `estimatesmartfee 2`? (Shouldn't the "2" be configurable?)

Reply:
'2' is the smallest you can ask bitcoind for, ie: I really want this now.